### ISSUE TITLE
Synchronized Media does not have an audio description button

### ISSUE THREAD

**Issue by jill-iterators** [CityOfBoston/Iterators#137]
Screengrab/videos (insert file names here)
2020 08 21, Synchronized Media does not have an audio description button.mp4

Link
https://d8-stg.boston.gov/civic-engagement/love-your-block

Node # (if known)
2861

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 10.15.6 > Chrome 84

Trusted Tester Test
17) Synchronized media

Brief Description
The video on the “My Brothers Keeper” article page does not have an Audio Description button

Steps to reproduce
1)	Go to the “Love Your Block article” and find the video on the page
2)	00:00 Check out the features that are available for the video and compare it to the features that are needed for the video to be Accessible, such as an Audio Description buttons. 

Desired Result
•	The videos on the page have all the required features for the synchronized media to be accessible

Actual Result
•	The videos on the page do not have: an Audio Description button
[2020 08 21, Synchronized Media does not have an audio description button.mp4.zip](https://github.com/CityOfBoston/Iterators/files/5111125/2020.08.21.Synchronized.Media.does.not.have.an.audio.description.button.mp4.zip)

**Comment by subaha-cob** [comment]
@andrebtlima Same as this issue "Synchronized Media Accessibility Issues #136" This will need work and documentation so that others can do the same when adding videos.

**Comment by subaha-cob** [comment]
@andrebtlima I will need access to our youtube channel to fix this. Otherwise, we can assignment to the content owner to fix. A good example of a page that passes the accessibility test is: https://d8-stg.boston.gov/civic-engagement/my-brothers-keeper-city-boston

**Comment by david-iterators** [comment]
Note:  These related issues should consider all of the various Synchronized Media requirements:  1.2.1 Captions (Prerecorded), 1.2.3 Audio Descriptions or Media Alternative (Prerecorded),  1.2.5 Audio Description (Prerecorded)
